### PR TITLE
ci: Replace make test-intergration within ACN PR pipeline

### PR DESCRIPTION
**Pull request by jpayne3506**
**Reason for Change**:

Replacing `make test-integration` with `make test-load` to unify the testing being done on CNIv2 E2E. In doing this goldpinger test are being removed from the PR pipeline with the goal of using them within the CNI Release Test Pipeline.

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [x] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by pjohnst5 — DISMISSED** [review]
Let's GOO! 🚀

**Comment by jpayne3506** [comment]
Breaking this PR into multiple PRs due to increased velocity from merge queue
https://github.com/Azure/azure-container-networking/pull/2480 - Increased VM Sku
https://github.com/Azure/azure-container-networking/pull/2481 - Verbose `make test-load`
https://github.com/Azure/azure-container-networking/pull/2482 - Dual OS E2E step template improvements
https://github.com/Azure/azure-container-networking/pull/2483 - Improve `RestartCNSDaemonset()` reliability
https://github.com/Azure/azure-container-networking/pull/2484 - Improve TestValidateState
https://github.com/Azure/azure-container-networking/pull/2485 - Improve Validate package

**Comment by jpayne3506** [comment]
Once #2484 and #2485 merge, this PR will be rebased and ready for review.

File: .pipelines/singletenancy/aks-swift/e2e-step-template.yaml [hunk 1]
```diff
@@ -52,29 +52,13 @@ steps:
       pwd
       kubectl cluster-info
       kubectl get po -owide -A
-      sudo -E env "PATH=$PATH" make test-integration CNS_VERSION=$(make cns-version) CNI_VERSION=$(make cni-version) INSTALL_CNS=true INSTALL_AZURE_VNET=true
+      sudo -E env "PATH=$PATH" make test-load SCALE_UP=32 OS_TYPE=linux CNI_TYPE=cniv2 VALIDATE_STATEFILE=true INSTALL_CNS=true INSTALL_AZURE_VNET=true CNS_VERSION=$(make cns-version) CNI_VERSION=$(make cni-version) CLEANUP=true
```
Comment (vipul-21) [inline .pipelines/singletenancy/aks-swift/e2e-step-template.yaml:55]: Curious to know, why `32` ?

**Review by vipul-21 — APPROVED** [review]

**Review by pjohnst5 — APPROVED** [review]
Nice work John!

File: .pipelines/singletenancy/aks-swift/e2e-step-template.yaml (hunk 1 above)
Comment (jpayne3506) [reply .pipelines/singletenancy/aks-swift/e2e-step-template.yaml:55]: Arbitrary number larger then 2x batch size.

**Pull request merged**